src/TopLevelDomain/CountryCodeTLD.php

```diff
@@ -5,4 +5,305 @@
 
 enum CountryCodeTLD: string
 {
+    case  = 'ලංකා';
```

Comment:
We should throw an exception on empty keys.

Reply:
Agreed! You can create a PR if you want? Otherwise I'll create one. ;)

Reply:
I'll need to dive in the underlying issue where Singhalan is not transliterated at all.

Reply:
👀 https://unicode.org/charts/PDF/U0D80.pdf

Reply:
![image](https://github.com/PrinsFrank/standards/assets/952007/5a752682-f704-4f2b-94ff-67d367f344a5)

https://aksharamukha.appspot.com/converter